src/Views/MainPage/components/Content/Input/Input.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { PureComponent } from 'react';
+import { withStyles } from '@material-ui/core';
+import TextField from '@material-ui/core/TextField';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Grid from '@material-ui/core/Grid';
+import styles from './Input.style';
+
+class Input extends PureComponent {
+  state = {
+    error: false,
+  }
+
+  losesFocus = () => {
+    const { input: { isReady, touched } } = this.props;
+    if (touched === false && isReady === false) {
```

Comment:
Co więcej, jak już poprawisz warunek, to możesz go wykorzystać w zupełnie inny sposób. Zamiast opakowywać to w ifa i else'a, możesz zrobić jeden `setState`. Powiedzmy: `this.setState({ key: <condidtion> ? value1 : value2 })` a w Twoim przypadku `this.setState({ error: !touched && !isReady })` by wystarczyło